styles/rentry.co/catppuccin.user.css

```diff
@@ -0,0 +1,558 @@
+      &.danger > .admonition-title::before {
+        filter: none !important;
+      }
+    }
+    // code
+    .highlighttable {
+      border-left-color: @surface0 !important;
+    }
+    .linenodiv > pre > .normal > a:not(:hover),
+    .linenodiv > pre > a:not(:hover) {
+      color: @overlay2 !important;
+    }
+    .entry-text pre,
+    .markdownx-preview pre,
+    code {
+      color: @red !important;
+    }
+    .highlight {
+      .n,
+      .nx {
+        color: @text;
+      }
+      .hll {
+        background-color: @yellow;
+      }
+      // Braces, Delimiters
+      .p {
+        color: @overlay2;
+      }
+      // Classes, Interfaces, Annotations, Metadata, Enums, Types
+      .cp {
+        color: @yellow;
+      }
+      .gd {
+        color: @red;
+      }
+      .gr {
+        color: @red;
+      }
+      .gi {
+        color: @green;
+      }
+      .go {
+        color: @overlay0;
+      }
+      .gu {
+        color: @mauve;
+      }
+      .gt {
+        color: @sapphire;
+      }
+      // Attributes
+      .na {
+        color: @yellow;
+      }
+      .no {
+        color: @yellow;
+      }
+      .ni {
+        color: @subtext0;
+      }
+      .ne {
+        color: @peach;
+      }
+      .nl {
+        color: @yellow;
+      }
+      .nn {
+        color: @text;
+      }
+      .w {
+        color: @subtext1;
+      }
+      // Escape Sequences,  Regex
+      .se {
+        color: @pink;
+      }
+      // Comments
+      .c,
+      .ch,
+      .cm,
+      .c1,
+      .cs {
+        color: @overlay2;
+      }
+      // Keyword
+      .k,
+      .kc,
+      .kd,
+      .kn,
+      .kp,
+      .kr,
+      .nt,
+      .sx,
+      .bp,
+      .kt {
+        color: @mauve;
+      }
+      // Operators
+      .o {
+        color: @sky;
+      }
+      // Constants, Numbers
+      .m,
+      .mb,
+      .mf,
+      .mh,
+      .mi,
+      .mo,
+      .il,
+      .nb {
+        color: @peach;
+      }
+      .gh,
+      .gp {
+        color: @sapphire;
+      }
+      // Strings
+      .s,
+      .sa,
+      .sb,
+      .sc,
+      .dl,
+      .s2,
+      .sh,
+      .sd,
+      .s1,
+      .cpf {
+        color: @green;
+      }
+      .nc {
+        color: @yellow;
+      }
+      // Methods, Functions
+      .nf,
+      .fm {
+        color: @blue;
+      }
+      .nd,
+      .ow {
+        color: @mauve;
+      }
+      .nv,
+      .ss,
+      .vc,
+      .vg,
+      .vi,
+      .vm {
+        color: @text;
+      }
+      .si,
+      .sr {
+        color: @red;
+      }
+      pre {
+        color: @text !important;
+      }
+      border-color: @surface0 !important;
+    }
+    // table
+    .mtable > tbody > tr:nth-child(even) {
+      background-color: @mantle !important;
+    }
+    .ntable th {
+      background-color: @surface1 !important;
+      border-color: @surface2 !important;
+      color: @text;
+    }
+    .ntable tr:nth-child(even) {
+      background-color: @surface0 !important;
+    }
```

Comment:
As a reader it's hard to tell what the difference between mtable and ntable is. Can you comment it? Is it a typo?

Reply:
this is not a typo. it seems like the `.mtable` class is only used in the big table in the "how" tab when editing. that one table is different than other tables. and i made its `tr:nth-child(even)` darker (mantle) because the example of a table happens to be on an even row which will make the even and odd rows the same color (the odd rows take their color from the parent). this is a specific case that only exists in the how tab (afaik you can't as a user make a table inside a table)
i'm not sure how i should comment it. should i go with "'HOW' tab table"?

Reply:
> i'm not sure how i should comment it. should i go with "'HOW' tab table"?

Yeah that sounds good!